serial_netbsd.go

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright 2014-2023 Cristian Maglie. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+
+package serial
+
+import "golang.org/x/sys/unix"
+
+const devFolder = "/dev"
+const regexFilter = "(dty|tty|dtyU|ttyU)[0-9]{1,2}"
```

Comment:
What's the difference between `dty` and `tty`?

Reply:
> What's the difference between `dty` and `tty`?

From man tty(4) on NetBSD:
The /dev/ttyXX special file is used for dial-in modems and terminals.
The /dev/dtyXX special file is a SunOS-compatible dial-out device.

Reply:
From man tty(4) on NetBSD:
The /dev/ttyXX special file is used for dial-in modems and terminals.
The /dev/dtyXX special file is a SunOS-compatible dial-out device.